django/db/models/options.py

```diff
@@ -545,6 +545,8 @@ def get_field(self, field_name):
         """
         Return a field instance given the name of a forward or reverse field.
         """
+        if field_name == 'pk':
+            return self.pk
```

Comment:
I just find it a bit odd that `get_field()` now returns `None` in cases where `self.pk is None` while it raises `FieldDoesNotExist` when other fields are missing.